components/DetailPage/SideBtn.tsx

```diff
@@ -46,13 +48,14 @@ const SideBtn = () => {
     method: 'patch',
     url: `/club/${clubDetail.id}/${clubDetail.isOpened ? 'close' : 'open'}`,
     errors: '동아리 열기/닫기에 실패했습니다',
+    autoPushToggle: false,
     onSuccess: () => {
       dispatch(setIsOpened())
     },
   })
 
   const handleAplly = () => {
```

Comment:
```suggestion
  const handleApply = () => {
```
hotfix이긴 하나 오타 잡아주면 좋긴 해 👍

Reply:
이건 걍 vscode에서 수정하는게 낫겠다